spec/models/order_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  # pending "add some examples to (or delete) #{__FILE__}"
+
+  subject { Order.new( product_name: "gears", product_count: 7, customer: FactoryBot.create(:customer) ) }
+  it "is valid with valid attributes" do
+    expect(subject).to be_valid
+  end
+  it "is not valid if no customer is selected" do
+    subject.customer=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid if it does not have a customer first name" do
```

Comment:
I think test cases about customer's name are not relevant here. You can't create a customer without name, therefore you can't create order if there's no customer.